index.js

```diff
@@ -1,8 +1,8 @@
 'use strict'
 
-const npm = require('npm')
```

Comment:
No need of npm in package.json any more.

Reply:
I thought of removing it but had the concern that it's hard to know what version of nom users will have.

By installing it ourselves we won't be caught off guard by breaking changes in the cli.
This PR uses the locally installed npm for this reason.